library/lib/javascript.js

```diff
@@ -1,26 +1,36 @@
 /* global __webpack_js_chunk_information__, __webpack_public_path__ */
 
-export default getSharedChunkFileNames(__webpack_js_chunk_information__).map((filename, index) =>
-  <script key={`javascript_${index}`} defer src={__webpack_public_path__ + filename} />
-)
+const { universalChunkNames, manifest } = __webpack_js_chunk_information__
 
-function getSharedChunkFileNames({ universalChunkNames, manifest }) {
+export default getSharedScriptsForAll(universalChunkNames)
+
+export function getSharedScriptsFor(name) {
+  return getSharedScriptsForAll([name])
+}
+
+export function getSharedScriptsForAll(names) {
   const sharedChunks = []
 
-  universalChunkNames.forEach(universalChunkName => {
-    addDependencies(manifest[universalChunkName].dependencies)
+  names.forEach(name => {
+    const info = manifest[name]
+    if (!info) throw new Error(`Could not find information in manifest for ${name}`)
+    addChunk(info)
   })
 
-  return sharedChunks
+  return sharedChunks.map(toScripts)
 
-  function addDependencies(dependencies) {
-    dependencies.map(x => manifest[x]).forEach(addFilenames)
+  function addChunk ({ dependencies = [], filename }) {
+    dependencies.sort((a, b) => b === 'runtime' ? 1 : 0).map(x => manifest[x].filename).forEach(addFilename)
```

Comment:
This is not good. Please use the property from the manifest entry. We are not guaranteed it will be called `runtime`.